actionview/test/template/form_helper/form_with_test.rb

```diff
@@ -353,6 +353,14 @@ def test_form_with_requires_arguments
     assert_equal "First argument in form cannot contain nil or be empty", error.message
   end
 
+  def test_form_with_when_given_nil_model_argument
+    error = assert_raises(ArgumentError) do
+      form_with(model: nil) do
+      end
+    end
+    assert_equal "Form model object is nil", error.message
```

Comment:
This needs to be changed now as well ;)

Reply:
lol thanks, I was eating lunch and thinking more about my food clearly hahah sorry

Reply:
@p8 - Ok, that's been updated as well 🤦‍♂️

Reply:
Thanks! 🍔